From 925dbb9edc9ee879adb13a144411eb7d65b37a47 Mon Sep 17 00:00:00 2001
From: Zhangbin Tong <zebulun.tong@rock-chips.com>
Date: Thu, 5 Nov 2020 10:19:23 +0800
Subject: [PATCH] ARM: dts: rv1126: Add shared dma memory pool inactive
 optional property

The ISP and CMA need the operating system must not active the region.

Signed-off-by: Zhangbin Tong <zebulun.tong@rock-chips.com>
Change-Id: Id27f453f8f1734d992d0e5ae9c38a01b0f7686fe
---
 arch/arm/boot/dts/rv1126.dtsi | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/arch/arm/boot/dts/rv1126.dtsi b/arch/arm/boot/dts/rv1126.dtsi
index 7d40e3384f39..5476b33ae95d 100644
--- a/arch/arm/boot/dts/rv1126.dtsi
+++ b/arch/arm/boot/dts/rv1126.dtsi
@@ -316,6 +316,14 @@
 		#size-cells = <1>;
 		ranges;
 
+		linux,cma {
+			compatible = "shared-dma-pool";
+			inactive;
+			reusable;
+			size = <0x800000>;
+			linux,cma-default;
+		};
+
 		drm_logo: drm-logo@00000000 {
 			compatible = "rockchip,drm-logo";
 			reg = <0x0 0x0>;
@@ -323,6 +331,7 @@
 
 		isp_reserved: isp {
 			compatible = "shared-dma-pool";
+			inactive;
 			reusable;
 			size = <0x10000000>;
 		};
-- 
2.35.3

